integrationExamples/gpt/pbjs_example_gpt.html

```diff
@@ -104,6 +104,13 @@
                         }
                     },
                     {
+                        bidder: 'adgear',
+                        // optional params: [ 'url', 'sizes' ]

```

Comment:
I added some explanation in the comment for url and sizes. The url param is used to optionally override the default window.location.href as the 'declared url', typically for pubs who want to hide that from the ad server. The sizes param is used to optionally override the sizes in the adUnit -- typically used for pubs who have a setup where the adUnit asks for two compatible formats but for whatever reason in AdGear they've only got one of the two defined (e.g., just call for 300x250 instead of both 300x250 and 300x600 defined in the adUnit).